worker/src/DepLibUring.cpp

```diff
@@ -319,8 +329,24 @@ void DepLibUring::LibUring::StopPollingCQEs()
 	uv_close(reinterpret_cast<uv_handle_t*>(this->uvHandle), static_cast<uv_close_cb>(onCloseFd));
 }
 
+uint8_t* DepLibUring::LibUring::GetSendBuffer()
+{
+	MS_TRACE();
+
+	if (this->availableUserDataEntries.empty())
+	{
+		MS_WARN_DEV("no user data entry available");
```

Comment:
Should this really be a warning (despite it's a dev log). I think it's a perfectly legitimate case so a debug should be fine.